include/post_update.php

```diff
@@ -16,6 +16,9 @@ function post_update() {
 
 	if (!post_update_1198())
 		return;
+
+	if (!post_update_1206())
```

Comment:
[standards] Please add braces to this instance and retroactively to the three previous ones 😊